app/ui.js

```diff
@@ -314,4 +295,99 @@ let productPopPrompt = () => {
       resolve(results);
     })
   });
+};
+
+module.exports.displayOrder = (total) => {
+  console.log(`
+  ${magenta('**  Your order total is,' total, 'Ready to purchase (Y/N)  **')}`);
+    prompt.get([{
+      name: 'choice',
+      description: 'Please make a selection'
+    }], orderMenuHandler );
+};
+
+let orderMenuHandler = (err, userInput) => {
+  console.log("user input", userInput);
+  // This could get messy quickly. Maybe a better way to parse the input?
+  if(userInput.choice == 'Y') {
+      completeOrderPrompt()
+    .then( (completeOrder) => {
+      console.log('this order is completed:', completeOrder);
+      //run post payment to order function
+    });
+  } else if (userInput.choice == 'N'){
+    activeCustomerPrompt()
+    .then( (activeCustomer) => {
+      console.log('this customer is now active:', activeCustomer)
+      //run active customer function that opens the customerMenuHandler
+    });
+  } else if (userInput.choice == '3') {
+    prompt.stop();
+  }
+};
+
+let completeOrderPrompt = () => {
```

Comment:
Change the name of this and where it's called-- I keep getting this error: Identifier 'completeOrderPrompt' has already been declared